policyengine_us/parameters/gov/states/ky/tax/income/deductions/itemized.yaml

```diff
@@ -0,0 +1,16 @@
+description: Kentucky allows these itemized deductions.
+values:
+  2022-01-01:
+    - home_mortgage_interest
+    - investment_income_form_4952
+    - charitable_cash_donations
+    - charitable_non_cash_donations
+    - gambling_losses_capped_at_winnings
+metadata:
+  unit: variable
+  label: Kentucky itemized deductions
+  reference:
+    - title: Kentucky Revised Statutes 141.019 (2)(i)
```

Comment:
Sorry for not noticing this sooner, but I think we should follow the legal code here instead, which specifies federal itemized deductions that _don't_ apply. This would more accurately capture the full set of deductions with respect to the federal rules.